backend/src/models/projects/budget.js

```diff
@@ -117,10 +117,32 @@ const findProjectBudgetByFiscal = (projectId) => {
     .groupBy("fy.fiscal_year");
 };
 
+const findPortfolioBreakdown = (projectId) => {
+  return knex
+    .select(
```

Comment:
Not a dealbreaker, but just FYI we have been leaning towards using the standard of:
```
.select({
    select_alias_1: "table.select_column"),
    select_alias_2: knex.raw("some complicated query"),
})
```
Instead of
```
.select(
"table.column AS select_alias_1",
knex.raw("some complicated query AS select_alias_2")
)
```